test/e2e/scaling_dbsize_test.go

```diff
@@ -0,0 +1,66 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/openshift/cluster-etcd-operator/pkg/etcdenvvar"
+	"github.com/openshift/cluster-etcd-operator/test/e2e/framework"
+	"github.com/openshift/library-go/test/library"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	operandNameSpace  = "openshift-etcd"
+	etcdCRName        = "cluster"
+	etcdLabelSelector = "app=etcd"
+	quotaEnvName      = "ETCD_QUOTA_BACKEND_BYTES"
+	etcdPodName       = "etcd"
+)
+
+func TestEtcdDBScaling(t *testing.T) {
+	dbSize32GB := int32(32)
+	opClientSet := framework.NewOperatorClient(t)
+	clientSet := framework.NewClientSet("")
+	etcdPodsClient := clientSet.CoreV1Interface.Pods(operandNameSpace)
+
+	etcdCR, err := opClientSet.OperatorV1().Etcds().Get(context.Background(), etcdCRName, metav1.GetOptions{})
+	require.NoError(t, err)
+
+	etcdCR.Spec.BackendQuotaGiB = dbSize32GB
+	etcdCR, err = opClientSet.OperatorV1().Etcds().Update(context.Background(), etcdCR, metav1.UpdateOptions{})
+	require.NoError(t, err)
+
+	err = library.WaitForPodsToStabilizeOnTheSameRevision(t, etcdPodsClient, etcdLabelSelector, 5, 1*time.Minute, 5*time.Second, 30*time.Minute)
```

Comment:
@tjungblu like this?

Reply:
yes, even though the 5s polling seems a bit too crazy, but it's alright